src/lazy/encoder/binary/v1_1/value_writer.rs

```diff
@@ -670,7 +670,7 @@ impl BinaryValueWriter_1_1<'_, '_> {
 
     fn eexp_writer<'a>(
         self,
-        macro_id: impl Into<MacroIdRef<'a>>,
+        macro_id: impl MacroIdLike<'a>,
```

Comment:
🪧 This trait is defined later, in `macro_table.rs`. Things that implement this are either identifiers (a `&str` name, a `usize` address, a `MacroIdRef` enum) or a resolved `&Macro` that can provide its identifier for serialization.